tests/test_rpc.py

```diff
@@ -1348,3 +1345,249 @@ def test_gethistory(revault_network, bitcoind, executor):
             )
             == 5
         )
+
+
+def get_unvault_txids(wallet, vaults):
+    unvault_txids = []
+    for vault in vaults:
+        deposit = f"{vault['txid']}:{vault['vout']}"
+        unvault_psbt = serializations.PSBT()
+        unvault_b64 = wallet.rpc.listpresignedtransactions([deposit])[
+            "presigned_transactions"
+        ][0]["unvault"]
+        unvault_psbt.deserialize(unvault_b64)
+        unvault_psbt.tx.calc_sha256()
+        unvault_txids.append(unvault_psbt.tx.hash)
+    return unvault_txids
```

Comment:
Is it a good practice to import functions from other test files? This function is present in `test_misc.py`. I can also move it into a different file. Let me know what would be the best approach

Reply:
I would make it a method of the `RevaultNetwork` class.